src/system/version_migration/versions/v1_1_0.ts

```diff
@@ -0,0 +1,29 @@
+import { SessionSaveData } from "../../game-data";
+import { CustomPokemonData } from "#app/data/custom-pokemon-data";
+
+export const systemMigrators = [] as const;
+
+export const settingsMigrators = [] as const;
+
+export const sessionMigrators = [
+  /**
+   *  Converts old Pokemon natureOverride and mysteryEncounterData
+   *  to use the new conjoined {@linkcode Pokemon.customPokemonData} structure instead.
+   *  @param data {@linkcode SessionSaveData}
+   */
+  function migrateCustomPokemonDataAndNatureOverrides(data: SessionSaveData) {
+    // Fix Pokemon nature overrides and custom data migration
+    data.party.forEach(pokemon => {
+      if (pokemon["mysteryEncounterData"]) {
+        pokemon.customPokemonData = new CustomPokemonData(pokemon["mysteryEncounterData"]);
+      }
+      if (pokemon["fusionMysteryEncounterData"]) {
+        pokemon.fusionCustomPokemonData = new CustomPokemonData(pokemon["fusionMysteryEncounterData"]);
```

Comment:
```suggestion
      if (pokemon["mysteryEncounterPokemonData"]) {
        pokemon.customPokemonData = new CustomPokemonData(pokemon["mysteryEncounterPokemonData"]);
      }
      if (pokemon["fusionMysteryEncounterPokemonData"]) {
        pokemon.fusionCustomPokemonData = new CustomPokemonData(pokemon["fusionMysteryEncounterPokemonData"]);
```

I think this is what these should be?
However due to how session migration is setup I don't think either of these options will work, see [my attempt at explaining why on discord](https://discord.com/channels/1125469663833370665/1296876320009093160)